tests/test_definitions/column_map_expectations/expect_column_values_to_match_regex_list.json

```diff
@@ -100,7 +100,7 @@
         "unexpected_index_list": [],
         "success": true
       },
-      "suppress_test_for": ["sqlite", "mssql"]
+      "suppress_test_for": ["sqlite", "mssql", "bigquery"]
```

Comment:
Bigquery does not allow for column names to contain spaces, which is what this test tests. Test is therefore supressed